deploy/examples/operator-openshift.yaml

```
@@ -112,6 +113,8 @@ data:
   ROOK_CSI_ENABLE_CEPHFS: "true"
   # Enable the default version of the CSI RBD driver. To start another version of the CSI driver, see image properties below.
   ROOK_CSI_ENABLE_RBD: "true"
+  # Enable the default version of the CSI CEPHNFS driver. To start another version of the CSI driver, see image properties below.
+  # ROOK_CSI_ENABLE_CEPHNFS: "true"
```

For visibility I'd suggest this setting be uncommented and set to false, even though it's the same default this way.
```suggestion
  ROOK_CSI_ENABLE_CEPHNFS: "false"
```